README.md

```diff
@@ -142,6 +150,10 @@ Filter the maximum number of days to keep real time stats available for. The def
 
 Insights and aggregated analytics data can be calculated, updated and stored in the database in cases where you wish to retain information for longer periods of time such as number of page views.
 
+**`altis.analytics.max_s3_backup_age <int>`**
+
+Filter the maximum number of days to keep backup data for. The default number of days is 90 in accordance with AWS Pinpoint, after which time data is removed. This is important for streamlining your user's privacy. There is no upper limit on this value however you should make sure any long term data storage is explained to users when opting in to tracking.
```

Comment:
The comma after AWS Pinpoint feels weird. I think it should either be removed, or another comma should be added after `90`.

`user's` should be `users'` to show plural user possessive.

& there should be a comma after `however`.

```suggestion
Filter the maximum number of days to keep backup data for. The default number of days is 90, in accordance with AWS Pinpoint, after which time data is removed. This is important for streamlining your users' privacy. There is no upper limit on this value, however you should make sure any long term data storage is explained to users when opting in to tracking.
```